plaso/containers/artifacts.py

```diff
@@ -219,6 +219,176 @@ def IsEquivalent(self, other):
     return False
 
 
+class PathArtifact(ArtifactAttributeContainer):
+  """Path artifact attribute container.
+
+  Attributes:
+    data_stream (str): name of a data stream.
+    path_segment_separator (str): path segment separator.
+    path_segments (list[str]): path segments.
+  """
+  CONTAINER_TYPE = 'path'
+
+  def __init__(self, data_stream=None, path=None, path_segment_separator='/'):
```

Comment:
Should the default path segment separator be OS independant?

Reply:
How do you see/imagine this? My thoughts were to set this on a per file system level or to maybe set this in the output.

Reply:
I have no idea. If this is just used internally by Plaso then I guess it doesn't really matter, I just wanted to point this out in case it was overlooked